graphql/resolvers.go

```diff
@@ -3101,6 +3101,21 @@ func (r *versionResolver) ChildVersions(ctx context.Context, v *restModel.APIVer
 		for _, cp := range childPatchIds {
 			v, err := r.Query().Version(ctx, cp)
 			if err != nil {
+				//before erroring due to the version being nil or not found,
+				// fetch the child patch to see if it's activated
+				p, err := patch.FindOneId(cp)
+				if err != nil {
+					return nil, err
+				}
+				if p == nil {
+					return nil, gimlet.ErrorResponse{
```

Comment:
This should be a graphql resolver error not a gimlet error.

Reply:
thanks! copy pasta, lol

Reply:
done